setup.py

```diff
@@ -40,7 +40,7 @@ def _get_version():
 def _get_data_files():
     data_files = [
         ("/etc/ubuntu-advantage", ["uaclient.conf"]),
-        ("/usr/lib/ubuntu-advantage", ["upgrade-lts-contract.py"]),
+        ("/usr/lib/ubuntu-advantage", glob.glob("lib/[!_]*")),
```

Comment:
Just tested package build and globbing does package this script and deliver it 
/usr/lib/ubuntu-advantage/upgrade_lts_contract.py. So, I think we are good here.